Global.h

```diff
@@ -745,6 +743,13 @@ namespace vstd
 		}
 		return ret;
 	}
+
+	STRONG_INLINE
+	ui32 getSid32(ui32 high, ui32 low)
+	{
+		return (high << 16) + low;
+	}
+	using boost::math::round;

```

Comment:
- Please do not change `Global.h` with no good reason. 
- If you really want this as a separate finction, add a method to `Bonus`